RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -4,7 +4,50 @@
 //  Copyright © yagom academy. All rights reserved.
 // 
 
-import Foundation
+enum RockScissorsPaperError: Error {
+    case invalidRockScissorsPaper
+    case invalidMukjippa
+}
 
-print("Hello, World!")
+enum RockScissorsPaperType: Int, CaseIterable {
+    case rock = 1
+    case scissors
+    case paper
+}
 
+enum RockScissorsPaperScenario {
+    case userWin
+    case userLose
+    case draw
+}
+
+var computerChoice: RockScissorsPaperType?
+var endFlag = false
+
+func printRockScissorsPaper() {
+    print("가위(1), 바위(2), 보(3)! <종료 : 0> : ", terminator: " ")
+}
+
+func createRandomRockScissorsPaper() -> RockScissorsPaperType? {
+    let choice = RockScissorsPaperType(rawValue: Int.random(in: 1...3))
+    guard let randomChoice = choice else {
+        return nil
+    }
+    return randomChoice
+}
```

Comment:
```suggestion
func createRandomRockScissorsPaper() -> RockScissorsPaperType {
    let choice = RockScissorsPaperType(rawValue: Int.random(in: 1...3))
    guard let randomChoice = choice else {
        return 
    }
    return randomChoice
}
```
초기에는 원시값을 사용하여 코드를 작성했습니다.